### PR TITLE
HPCC-14801 Document LDAP Feature permissions for DESDL

### PR DESCRIPTION
**Pull request by JamesDeFabia**
Signed-off-by: Jim DeFabia jamesdefabia@lexisnexis.com
@rpastrana Please review

### CODE REVIEW

**Comment by rpastrana** [comment]
Image looks reasonable

File: docs/DynamicESDL/DynamicESDL_Includer.xml [hunk 1]
```diff
@@ -129,22 +129,35 @@
...
           configuration XML file.</para>
 
-          <para />
+          <para></para>
         </listitem>
       </itemizedlist>
+
+      <para><emphasis role="bold">DESDL and LDAP Security</emphasis></para>
+
+      <para>If your HPCC platform is configured to use for LDAP security, you
```
Comment (rpastrana) [inline docs/DynamicESDL/DynamicESDL_Includer.xml]: "configured to use for LDAP" doesn't make sense.
Also, keep in mind that in the future we won't be 100% reliant on LDAP for authorization

**Comment by rpastrana** [comment]
@JamesDeFabia one comment

**Comment by JamesDeFabia** [comment]
@rpastrana Edits made, please have another look

**Comment by rpastrana** [comment]
@

**Pull request merged**